committeeoversightapp/models.py

```diff
@@ -187,14 +197,51 @@ class CategoryDetailPage(Page):
          icon='site'))
     ])
 
-    # Editor configuration
+    def get_url_parts(self, *args, **kwargs):
```

Comment:
Where would this be used?

Reply:
This customizes the routes for the detail pages. https://docs.wagtail.io/en/v2.6.1/topics/pages.html?highlight=get_url_parts#customising-url-patterns-for-a-page-model